lib/external_doc.rb

```diff
@@ -2,18 +2,18 @@
 require "uri"
 
 class ExternalDoc
-  def self.parse(markdown, repo_name: "", path: "")
+  def self.parse(markdown, repo_name: "", branch: "", path: "")
```

Comment:
maybe we can default to `master` and only need to pass in `main` if we need to?

Reply:
ExternalDocs is alwayas called from GithubRepoFetcher docs, so I made the change there.